src/ClientHandle.cpp

```diff
@@ -388,25 +392,30 @@ void cClientHandle::FinishAuthenticate(const AString & a_Name, const cUUID & a_U
 
 
 
-bool cClientHandle::StreamNextChunk(void)
+void cClientHandle::StreamNextChunks(void)
 {
 	ASSERT(m_Player != nullptr);
 
 	int ChunkPosX = m_Player->GetChunkX();
 	int ChunkPosZ = m_Player->GetChunkZ();
+
 	if ((m_LastStreamedChunkX == ChunkPosX) && (m_LastStreamedChunkZ == ChunkPosZ))
 	{
 		// All chunks are already loaded. Abort loading.
-		return true;
+		return;
+	}
+	else
+	{
+		m_LastStreamedChunkX = 0x7fffffff;
+		m_LastStreamedChunkZ = 0x7fffffff;
```

Comment:
Why those magic numbers?

Reply:
I see the sense of those values, but an explanation would be nice. or a `#define INSANEVALUE 0x7fffffff`

Reply:
Now I see the big change - you invalidate the last sent chunk if we're not in it 💡